### PR TITLE
Replaces the countdown font with a monospaced one

### PR DESCRIPTION
**Pull request by jsangmeister**

### CODE REVIEW

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Do we want to have this configurable (like all other fonts)?

**Comment by emanuelschuetze** [comment]
> @emanuelschuetze Do we want to have this configurable (like all other fonts)?

Yes, this would be nice. @jsangmeister Can you do it? Maybe ask @FinnStutzenstein

**Comment by jsangmeister** [comment]
Done. @emanuelschuetze @FinnStutzenstein please check if it fits your needs

File: client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts [hunk 1]
```diff
@@ -235,6 +240,9 @@ export class MotionListComponent extends BaseListViewComponent<ViewMotion> imple
         this.configService
             .get<boolean>('motions_statutes_enabled')
             .subscribe(enabled => (this.statutesEnabled = enabled));
+        this.configService
+            .get<boolean>('motions_amendments_enabled')
+            .subscribe(enabled => (this.amendmentsEnabled = enabled));
```
Comment (tsiegleauq) [inline client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts:245]: although tiny, a PR/commit should only concern one bug or feature at a time

**Comment by jsangmeister** [comment]
> The default font is pretty ugly

Hey, I spent at least 2 minutes searching for a nice monospace font on Google fonts, don't invalidate my work!

File: client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts (hunk 1 above)
Comment (jsangmeister) [reply client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts:245]: Must have mixed up my branches, my bad

**Comment by jsangmeister** [comment]
Squashed and cleaned up.

**Comment by FinnStutzenstein** [comment]
@jsangmeister Please check travis
@emanuelschuetze Can you try this out?

**Comment by jsangmeister** [comment]
fixed travis

**Pull request merged**